interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -14,7 +14,16 @@ angular.module('adage.analyze.sampleBin', [
 
 .factory('SignatureSet', ['$resource', 'ApiBasePath',
   function($resource, ApiBasePath) {
-    return $resource(ApiBasePath + 'node/set/:ids/');
+    return $resource(
+      ApiBasePath + 'node/post_multiple/',
+      {},
+      {post: {
+        method: 'POST',
+        // Setting Content-Type is required. Django will not process the
+        // POST data the way Angular's defaults send it.
+        headers: {'Content-Type': 'application/x-www-form-urlencoded'}
```

Comment:
Actually... hmm. Maybe the simplest fix for the testing problem I hit would be to keep this `Content-Type` as form-urlencoded and actually encode the POST body that way. Let me try that.